.github/workflows/enos-run.yml

```diff
@@ -37,21 +38,22 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ${{ inputs.go-version }}
+      - name: Log Go version
+        run: |
+          echo "${{ inputs.go-version }}"
       - name: Set up Terraform
         uses: hashicorp/setup-terraform@v2
         with:
           # the terraform wrapper will break Terraform execution in enos because
           # it changes the output to text when we expect it to be JSON.
           terraform_wrapper: false
       - name: Import GPG key for Boundary pass keystore
-        if: matrix.test == 'cli_ui'
```

Comment:
if you need this step for both `cli_ui` and `e2e` scenarios but not for `smoke` you might be able to update this condition to `if: matrix.test != 'smoke'`

Reply:
Ryan has updated Enos- I will remove the condition altogether 👏